src/Components/animation/DisplayedElement.js

```diff
@@ -161,6 +161,27 @@ class DisplayedElement extends React.Component {
 }
 
 const _initializeStateFromProps = props => {
+  /* Get the element width */
+  var width = 0;
```

Comment:
For a variable that changes value, it's recommanded to use `let`

Reply:
Ok. So "var" is never used? Just "let" and "const"?

Reply:
I'm not an expert, but it seems to me that "let" and "const" are more predictable and less prone to doing errors in the code. (var has a global scope and hoisting)